ch2driver/pytpcc/drivers/nestcollectionsdriver.py

```diff
@@ -714,79 +877,120 @@ def loadTuples(self, tableName, tuples):
+                v1 = tuple[l]
+                if tableName == constants.TABLENAME_WAREHOUSE and columns[l] == "w_address":
+                    v1 = self.genNestedTuple(v, constants.TABLENAME_WAREHOUSE_ADDRESS)
+                elif tableName == constants.TABLENAME_DISTRICT and columns[l] == "d_address":
+                    v1 = self.genNestedTuple(v, constants.TABLENAME_DISTRICT_ADDRESS)
+                elif tableName == constants.TABLENAME_CUSTOMER and columns[l] == "c_name":
+                    v1 = self.genNestedTuple(v, constants.TABLENAME_CUSTOMER_NAME)
+                elif tableName == constants.TABLENAME_CUSTOMER and columns[l] == "c_extra":
+                    if self.schema == constants.CH2_DRIVER_SCHEMA["CH2PP"]:
+                        for i in range(0, self.customerExtraFields):
+                            val[columns[l]+"_"+str(format(i+1, "03d"))] = v1[i]
+                    continue
+                elif tableName == constants.TABLENAME_CUSTOMER and columns[l] == "c_addresses":
+                    v1 = []
+                    for clv in v:
+                        v1.append(self.genNestedTuple(clv, constants.TABLENAME_CUSTOMER_ADDRESSES))
+                        if self.schema == constants.CH2_DRIVER_SCHEMA["CH2P"]:
+                            break # Load only one customer address for CH2P
+                elif tableName == constants.TABLENAME_CUSTOMER and columns[l] == "c_phones":
+                    v1 = []
+                    for clv in v:
+                        v1.append(self.genNestedTuple(clv, constants.TABLENAME_CUSTOMER_PHONES))
+                        if self.schema == constants.CH2_DRIVER_SCHEMA["CH2P"]:
+                            break # Load only one customer phone for CH2P
+                elif tableName == constants.TABLENAME_ORDERS and columns[l] == "o_extra":
+                    if self.schema == constants.CH2_DRIVER_SCHEMA["CH2PP"]:
+                        for i in range(0, self.ordersExtraFields):
+                            val[columns[l]+"_"+str(format(i+1, "03d"))] = v1[i]
+                    continue
+                elif tableName == constants.TABLENAME_ORDERS and columns[l] == "o_orderline":
+                    v1 = []
+                    for olv in v:
+                        v1.append(self.genNestedTuple(olv, constants.TABLENAME_ORDERLINE))
+                elif tableName == constants.TABLENAME_ITEM and columns[l] == "i_extra":
+                    if self.schema == constants.CH2_DRIVER_SCHEMA["CH2PP"]:
+                        for i in range(0, self.itemExtraFields):
+                            val[columns[l]+"_"+str(format(i+1, "03d"))] = v1[i]
+                    continue
+                elif tableName == constants.TABLENAME_SUPPLIER and columns[l] == "su_address":
+                    v1 = self.genNestedTuple(v, constants.TABLENAME_SUPPLIER_ADDRESS)
+                elif isinstance(v1,(datetime)):
+                    v1 = str(v1)
```

Comment:
Can we group the `if` branches by table name to make things a bit clearer?

I.e.
```
if tableName == constants.TABLENAME_WAREHOUSE and columns[l] == "w_address":
    ...
elif tableName == constants.TABLENAME_DISTRICT and columns[l] == "d_address":
    ...
elif tableName == constants.TABLENAME_CUSTOMER:
    if columns[l] == "c_name":
        ...
    elif columns[l] == "c_extra":
        ...
    elif columns[l] == "c_addresses":
        ...
    elif columns[l] == "c_phones":
        ...
elif tableName == constants.TABLENAME_ORDERS:
    if columns[l] == "o_extra":
        ...
    elif columns[l] == "o_orderline":
        ...
elif tableName == constants.TABLENAME_ITEM and columns[l] == "i_extra":
    ...
elif tableName == constants.TABLENAME_SUPPLIER and columns[l] == "su_address":
    ...
elif isinstance(v1,(datetime)):
    ...
```

Reply:
Done.